xDripG5/G5Transmitter.swift

```diff
@@ -0,0 +1,265 @@
+//
+//  G5Sensor.swift
```

Comment:
Better to `git mv Transmitter.swift G5Transmitter.swift` to maintain the commit history.